### PR TITLE
2.5 Branching Strategy

### LINKED ISSUE #213: 2.5 - Process Branching Strategy

### PR DESCRIPTION
**Pull request by tobloef**

### CODE REVIEW

**Comment by Duckapple** [comment]
Would it be better to describe GitHub Flow first and then compare and describe afterwards?

**Comment by tobloef** [comment]
> Would it be better to describe GitHub Flow first and then compare and describe afterwards?

Fixed.

**Review by Duckapple — APPROVED** [review]
looks nice IMO

**Comment by tobloef** [comment]
Closes #213

**Review by Hjaltesorgenfrei — CHANGES_REQUESTED** [review]
When and how are we adding References.

File: report/25_Process-Branching-Strategy.md [hunk 1]
```diff
@@ -1 +1,12 @@
-Applied branching strategy.
+## Branching Strategy
+
+Due to the smaller scope and team size, it was decided to use a lightweight, but surprisingly useful, branching strategy called GitHub flow (TODO: ref) (not to be confused with Git flow). GitHub flow works like this:
```
Comment (Jakobis) [inline report/25_Process-Branching-Strategy.md]: Burde måske få ordnet din todo først?

**Review by Jakobis — DISMISSED** [review]

File: report/25_Process-Branching-Strategy.md (hunk 1 above)
Comment (Jakobis) [reply report/25_Process-Branching-Strategy.md]: @tobloef

**Review by Hjaltesorgenfrei — APPROVED** [review]

**Pull request merged**